src/garage/tf/algos/_utils.py

```diff
@@ -0,0 +1,31 @@
+"""Utility functions used by tensorflow algorithms."""
+
+
+class _Default:  # pylint: disable=too-few-public-methods
```

Comment:
The semantic of `_Default` is **_allowing_ a parameter to be missing and use default instead**.

In `make_optimizer()`, if a parameter in kwargs is not wrapped with `_Default`, and it also appears in `optimizer_type`, a `ValueError` should be raised.

See here https://github.com/rlworkgroup/garage/blob/3d9c1b27a39addb21c8df4fd6489c19eee5c6aeb/src/garage/torch/algos/_utils.py#L43